src/usr/local/buildpack/util.sh

```diff
@@ -68,6 +68,22 @@ function check_semver () {
   export PATCH=${BASH_REMATCH[5]}
 }
 
+function validate_semver () {
+  local number_of_parts=${1:-3}
```

Comment:
shouldn't we call `check_semvrr` here? so we've one less code line?

Reply:
yes, we can do that

Reply:
better pass version as parameter too, so this function doesn't need to know the env name